guides/common/modules/proc_registering-project-as-a-client-of-keycloak.adoc

```diff
@@ -63,6 +64,20 @@ Use `hammer-openidc` as the application name.
 --keycloak-auth-role root-admin \
 -t openidc -l /users/extlogin --force
 ----
+.. Configure {Project} to use {keycloak} as an authentication source for CLI:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# {foreman-installer} --foreman-keycloak true \
```

Comment:
```suggestion
# {foreman-installer} \
--foreman-keycloak true \
```

Reply:
I wonder why you're suggesting the line break. There are plenty of command blocks with lines that have even more characters than this one.